Orders/Orders.cpp

```diff
@@ -48,11 +48,14 @@ bool Negotiate::validate(){
     else if(Negotiate::isNegotiation(this->getPlayerIssuerID(), this->getPlayerTargetID())){return false;}
     else{return true;}
 }
-
+string Order::stringToLog() {
+    string out = "Executed the Order:" + description;

```

Comment:
make sure to dereference the description variable it’s a string pointer (here and anywhere else you use it for the logs)